src/main/java/dev/aurelium/auramobs/AuraMobs.java

```diff
@@ -129,6 +135,11 @@ public void loadWorlds() {
     public void registerCommands() {
         PaperCommandManager manager = new PaperCommandManager(this);
         manager.registerCommand(new AuraMobsCommand(this));
+        manager.getCommandCompletions().registerCompletion("entitytypes", c -> Arrays.stream(EntityType.values())
+                .filter(type -> type.isSpawnable() && type.isAlive())
```

Comment:
Additionally, filter out passive mobs using the method described in the comment on AuraMobsCommand